ClientRuntimes/CSharp/ClientRuntime.Azure.Authentication/UserTokenProvider.cs

```diff
@@ -261,33 +261,27 @@ public class UserTokenProvider : ITokenProvider
         public static async Task<ServiceClientCredentials> LoginWithPromptAsync(string domain, ActiveDirectoryClientSettings clientSettings, 
```

Comment:
@stankovski I think this PR is for updating ADAL to v3.5. Will be another PR including the nuspec, json and dnu build changes.